apps/web/src/server/app/lpa/lpa.service.js

```diff
@@ -46,3 +46,11 @@ export async function findAllIncomingIncompleteQuestionnaires() {
 
 	return questionnairesListData;
 }
+
+export async function findQuestionnaireById(id) {
+	const data = await request(`case-officer/${id}`);
+
+	data.AppealSiteHtml = data.AppealSite ? data.AppealSite.replaceAll(',', '<br />') : '';
```

Comment:
HTML manipulation shouldn't be in the service. You can just do this in nunjucks anyway, for example: {{ data.AppealSite | replace(",", "<br />")

Reply:
A good point! The data format for the AppealSite has changed since I wrote this originally, so it would have needed updating anyway. I've refactored to move this out of the service into the controller.